### ISSUE TITLE
Improve the `try-backdrop page`

### ISSUE THREAD

**Issue by klonos** [backdrop-ops/backdropcms.org#555]
@olafgrabienski in https://github.com/backdrop-ops/backdropcms.org/issues/422#issuecomment-498995105
> 1. On the Home page, under the "Demo Backdrop" button, there's the link for "other ways to try" Backdrop. When you click it, you arrive at the top of https://backdropcms.org/try-backdrop and see a description of our own demo, not of the "other ways". It would be better to arrive further down on the page, or to change the link text on the Home page, something with the meaning "all ways to try" (sorry for my English).

@jenlampton 
> I'm not sure about this, people will need to see the page title.
> 
> I want this page to serve two purposes - for the people who click the link on the home page, but also for people who land right on it. For the second group, they need to see the demo button first.
> 
> Maybe we should change the language from "other ways to try" to "more ways to try" so it doesn't seem like demo should be excluded? (done)

**Comment by klonos** [comment]
A few ideas for consideration:

1. Fix the Tugboat and DDEV logos to be transparent, like the rest.
2. Convert the gray headers into accordions, and have them be collapsed. That way, they'd all be visible when one lands on that page, and people can click on the one they're interested in to see details/steps.
3. If we don't want accordions, we can consider a plain bullet list or some sort of "in this page" index at the top, with links to each section.
4. Clearly denote, and perhaps also group by, local (lando/ddev) vs. permanent (pantheon/platform.sh) vs. temporary/disposable (tugboat).

**Comment by klonos** [comment]
> 3. If we don't want accordions, we can consider a plain bullet list or some sort of "in this page" index at the top, with links to each section.

Other ideas:
- jump list on the sidebar
- make the order of things random

**Comment by klonos** [comment]
Hey @jenlampton 👋 ...here are updated, transparent versions of the ddev and tugboat logos:

![ddev-logo](https://user-images.githubusercontent.com/2423362/62345214-104fce00-b4fa-11e9-9028-17c64265ed82.png)
![tugboat-logo](https://user-images.githubusercontent.com/2423362/62345215-10e86480-b4fa-11e9-8858-e4a9b2a420cd.png)

I have tried them on https://backdropcms.org/try-backdrop but they don't look as nice with that light gray (`#F7F7F7`) as background:

<img width="1175" alt="Screen Shot 2019-08-02 at 8 01 41 am" src="https://user-images.githubusercontent.com/2423362/62345653-d5e73080-b4fb-11e9-9e71-e0ef51cd3d22.png">

...here's a mockup if we used `#E8E6E5` as background instead:

<img width="1189" alt="Screen Shot 2019-08-02 at 7 59 24 am" src="https://user-images.githubusercontent.com/2423362/62345663-daabe480-b4fb-11e9-8ae4-1422e98d0fe9.png">

In the meantime, I have used "screencapture" versions of them instead of transparent:

<img width="201" alt="ddev-logo-screencapture" src="https://user-images.githubusercontent.com/2423362/62345471-3f1a7400-b4fb-11e9-9808-19b837b9989a.png">
<img width="164" alt="tugboat-logo-screencapture" src="https://user-images.githubusercontent.com/2423362/62345472-3fb30a80-b4fb-11e9-8bf0-b10c69b15579.png">

Let me know what you think.

**Comment by klonos** [comment]
...here's a backup of their old versions if you want to revert to those instead:

![ddev-logo-old](https://user-images.githubusercontent.com/2423362/62345786-48581080-b4fc-11e9-9e4c-20087fff3c0c.png)
![tugboat-logo-old](https://user-images.githubusercontent.com/2423362/62345787-48f0a700-b4fc-11e9-8b5d-4a976086e0f2.png)

**Comment by klonos** [comment]
...those screen-captured ones looked ugly too IMO:

<img width="1180" alt="Screen Shot 2019-08-02 at 8 14 57 am" src="https://user-images.githubusercontent.com/2423362/62346145-b3561700-b4fd-11e9-8d4d-2e4a4f4846b9.png">

...so here's another take on transparent ones:

![ddev-logo-transparent](https://user-images.githubusercontent.com/2423362/62346171-c963d780-b4fd-11e9-893a-01d3e10c8444.png)
![tugboat-logo-transparent](https://user-images.githubusercontent.com/2423362/62346173-c963d780-b4fd-11e9-8cc2-496a8c9ee146.png)

**Comment by klonos** [comment]
...yup, seems I was trying too hard 😅:

<img width="1177" alt="Screen Shot 2019-08-02 at 8 18 12 am" src="https://user-images.githubusercontent.com/2423362/62346267-30818c00-b4fe-11e9-971d-4983f0ec482b.png">

**Comment by klonos** [comment]
...I have also made the ordering of that view randomised 😉

**Comment by olafgrabienski** [comment]
@klonos Sorry, I didn't provide feedback before, but is a randomized order a good idea? In my opinion it's not user-friendly: When you use the page to find a preferred demo and return a few weeks later, you have to search 'your' demo again.

(Apart of that, it looks like due to caching the list isn't really random. Got the same order in two different browsers.)

I don't remember which was the order criteria before. What about an alphabetical list, corresponding to the organization names? (DDEV, Lando, ..., Tugboat)

Btw, I've found a typo in the DDEV section there is a double "local": `Create a local local Backdrop site on DDEV`

(While we're at it, there is also a typo on the 'Sandbox ready' page https://backdropcms.org/demo/created?url=... There is a space missing between "sandbox" and "is": `Note that sending email from demo sandboxis not allowed`.)

**Comment by klonos** [comment]
Sorry for taking too long to get back to you too @olafgrabienski ...I have fixed the double "local local" in the DDEV entry.

As for the random order, I believe that it was because we did not want it to seem as if we were endorsing one platform over the others. Being random gives all listed platforms a chance to be listed at the top.

As for why you got the same order twice, maybe that's because there's only a few platforms listed, so the randomness was randomly not random in your case 😆

**Comment by olafgrabienski** [comment]
Thanks for fixing the typo, @klonos! Regarding the order, I still think that the random one isn't very user-friendly but never mind.

**Comment by BWPanda** [comment]
FTR, I updated the Tugboat logo as they seem to have ditched the actual 'tugboat' image now.

**Comment by klonos** [comment]
FTR, I've added a new "Platform" node for Softaculous: https://backdropcms.org/platform/softaculous. The logo I found from https://www.softaculous.com/website/images/softaculous_logo.gif isn't as hi-res, so shows pixelated. We should ask them for a better one.

**Comment by bugfolder** [comment]
From a Zuliip discussion of the past few days:

@yorkshire-pudding suggested for this page:

> * Tugboat at the top
> * Backdrop 1.x branch last
> * all others alphabetically

@bugfolder suggested adding a `Weight` field to the Platform content type that would support sorting to push selected items to the top of bottom of the list (i.e., like 'sticky', but also providing 'anti-sticky').

@klonos suggested using https://backdropcms.org/project/draggableviews.

**Comment by klonos** [comment]
Yeah, although adding a custom weight field would work just fine, it's one of those cases where the field doesn't "belong" in the database as part of the content per se. Adding Draggable Views and basing our solution on that would would allow content authors on the site to reorder the entires in that page via drag and drop, without having to edit weight fields individually. Plus, I believe that we'll be able to use the module in other similar scenarios in the future - it's just one of those utility modules that simply make life easier.

**Comment by klonos** [comment]
...an alternative would be to make individual content blocks for each one of these entries, and then people could reorder them via the layout builder UI.

**Comment by BWPanda** [comment]
This sounds awfully complicated. I think a simple weight (integer) field would work fine, especially since the ordering won't really change that much (at all).

Tugboat: -1
Default/all others: 0
Backdrop Dev: 1

**Comment by jenlampton** [comment]
The weight field sounds reasonable, I think that's my preference too. But
that field needs to be limited to admins/editors only (I think anyone can
add a platform?)

**Comment by yorkshire-pudding** [comment]
> But that field needs to be limited to admins/editors only (I think anyone can add a platform?)

I don't think anyone can add a platform.  I have some form of editor rights so I can update the Office Hours events and these are the ones I could add.

![image](https://user-images.githubusercontent.com/20357797/224760427-2dd2f6df-6eaa-4b17-a4a4-a476599e2cc7.png)

**Comment by jenlampton** [comment]
Oh good, weight field it is? :)

**Comment by bugfolder** [comment]
I've used [Field Permissions](https://backdropcms.org/project/field_permissions) elsewhere for the sort of admin limitation we're talking about for this `weight` field. It's not currently installed on b.org. Any objections to adding it, or suggestions for a better alternative?

**Comment by jenlampton** [comment]
I think we determined that protecting the field isn't necessary. We just need to create it and update the view :)

**Comment by jenlampton** [comment]
> "Try Backdrop CMS". Should this page title be "More ways to try Backdrop CMS"

This was an easy one, so I made this change :)
